wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1968,6 +1971,23 @@ class KotlinGenerator private constructor(
     }
   }
 
+  private fun addBuildClosure(type: MessageType, companionBuilder: TypeSpec.Builder, builderClassName: ClassName) {
```

Comment:
```suggestion
  /**
   * Adds a closure into the [companionBuilder] allowing the creation of an instance via the Kotlin
   * DSL.
   *
   * Example
   * ```
   * companion object {
   *   public inline fun build(body: Builder.() -> Unit): AllTypes = Builder().apply(body).build()
   * }
   * ```
   */
  private fun addBuildClosure(type: MessageType, companionBuilder: TypeSpec.Builder, builderClassName: ClassName) {
```